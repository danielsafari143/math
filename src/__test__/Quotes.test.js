import renderer from 'react-test-renderer';
import Quote from '../components/Quotes';

describe('Confirming display of quote', () => {
  it('quote displays', () => {
    const quoteDisplay = renderer.create(<Quote />).toJSON();
    expect(quoteDisplay).toMatchSnapshot();
  });
});
