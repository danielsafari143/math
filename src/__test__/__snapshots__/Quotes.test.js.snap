// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirming display of quote quote displays 1`] = `
<div
  className="calculate"
>
  <p
    className="quote"
  >
    Loading...
  </p>
</div>
`;
